CoreHistogramView/src/au/gov/asd/tac/constellation/views/histogram/BinCollection.java

```diff
@@ -57,6 +59,19 @@ public GraphElementType getElementType() {
     public Bin[] getBins() {
         return bins;
     }
+    
+    public Bin[] getSelectedBins() {
```

Comment:
Are you able to add a unit test for this function you added (don't need to worry about the rest of the class)